loadtests/Makefile

```diff
@@ -12,7 +12,7 @@ INSTALL = ARCHFLAGS=$(ARCHFLAGS) ./venv/bin/pip install
 .env.install:
 	virtualenv ./venv
 	$(INSTALL) pexpect >> loadtest.log && \
-	$(INSTALL) configparser==3.3.0r2 >> loadtest.log && \
+	$(INSTALL) configparser==3.3.0post2 >> loadtest.log && \
```

Comment:
It seems they have a bit of a naming issue. The latest version is 3.5.0b2, which is a beta, not sure if you want to use that instead?